### PR TITLE
lodash: Add _.reduce for LoDashExplicitObjectWrapper

### PR DESCRIPTION
**Pull request by zuzusik**

### CODE REVIEW

File: lodash/lodash.d.ts [hunk 1]
```diff
@@ -8539,6 +8539,21 @@ declare module _ {
...
             callback: MemoIterator<TValue, TResult>): TResult;
     }
+    
+    interface LoDashExplicitObjectWrapper<T> {
+        /**
+        * @see _.reduce
+        **/
+        reduce<TValue, TResult>(
+            callback: MemoIterator<TValue, TResult>,
+            accumulator: TResult): TResult;
```
Comment (zuzusik) [inline lodash/lodash.d.ts]: err, this should be LoDashExplicitArrayWrapper<TResult>, will update now

**Comment by vvakame** [comment]
We can't get response from authors.
please ping me again in 2 days. I'll merge this PR.

**Comment by zuzusik** [comment]
@vvakame ping

**Comment by zuzusik** [comment]
@vvakame should we merge this?

**Comment by vvakame** [comment]
It merged. sorry for late!

**Pull request merged**